# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsContentSink.h
# Commit: ca2681b0f792
# Full Hash: ca2681b0f79274712ad5f99e34075fecc6a8629c
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 11: Don't preload anchored links from early hints r=necko-reviewers,valentin
#   
#   They won't get passed to the PreloadService anyway. Resulting in an unnecessary
#   additional load
#   
# ==============================================================================

diff -r cbbdb4b4aaa9 -r ca2681b0f792 dom/base/nsContentSink.h
--- a/dom/base/nsContentSink.h	Fri Dec 02 16:15:56 2022 +0000
+++ b/dom/base/nsContentSink.h	Fri Dec 02 16:15:57 2022 +0000
@@ -115,7 +115,6 @@
   virtual void UpdateChildCounts() = 0;
 
   bool IsTimeToNotify();
-  bool LinkContextIsOurDocument(const nsAString& aAnchor);
 
  protected:
   nsContentSink();